.travis.yml

```diff
@@ -1,30 +0,0 @@
-language: ruby
```

Comment:
FYI: Looks like Travis CI doesn't work now (I don't known why), so I decided to remove it and use GitHub Actions only.